docs/source/deployment/databricks/databricks_ide_development_workflow.md

```diff
@@ -10,263 +16,91 @@ By working in your local environment, you can take advantage of features within
-conda create --name iris-databricks python=3.10
-```
-
-Once it is created, activate it:
-
-```bash
-conda activate iris-databricks
-```
-
-With your Conda environment activated, install Kedro and dbx:
-
-```bash
-pip install kedro dbx --upgrade
-```
+### Install Kedro and Databricks CLI in a new virtual environment
 
 ### Authenticate the Databricks CLI
 
-**Now, you must authenticate the Databricks CLI with your Databricks instance.**
-
-[Refer to the Databricks documentation](https://docs.databricks.com/en/dev-tools/cli/authentication.html) for a complete guide on how to authenticate your CLI. The key steps are:
-
-1. Create a personal access token for your user on your Databricks instance.
-2. Run `databricks configure --token`.
-3. Enter your token and Databricks host when prompted.
-4. Run `databricks fs ls dbfs:/` at the command line to verify your authentication.
-
-```{note}
-dbx is an extension of the Databricks CLI, a command-line program for interacting with Databricks without using its UI. You will use dbx to sync your project's code with Databricks. While Git can sync code to Databricks Repos, dbx is preferred for development as it avoids creating new commits for every change, even if those changes do not work.
-```
-
-### Create a new Kedro project
-
-Create a Kedro project with the `databricks-iris` starter using the following command in your local environment:
-
-```bash
-kedro new --starter=databricks-iris
-```
-
-Name your new project `iris-databricks` for consistency with the rest of this guide. This command creates a new Kedro project using the `databricks-iris` starter template.
-
- ```{note}
-If you are not using the `databricks-iris` starter to create a Kedro project, **and** you are working with a version of Kedro **earlier than 0.19.0**, then you should [disable file-based logging](https://docs.kedro.org/en/0.18.14/logging/logging.html#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
- ```
-
-### Create a Repo on Databricks
-
-Create a new Repo on Databricks by navigating to `New` tab in the Databricks workspace UI side bar and clicking `Repo` in the drop-down menu that appears.
-
-In this guide, you will not sync your project with a remote Git provider, so uncheck `Create repo by cloning a Git repository` and enter `iris-databricks` as the name of your new repository:
-
-![Create a new Repo on Databricks](../../meta/images/databricks_repo_creation.png)
+### Create a new Kedro Project
 
-### Sync code with your Databricks Repo using dbx
+## Create the Databricks Asset Bundles using `kedro-databricks`
 
-The next step is to use dbx to sync your project to your Repo.
+`kedro-databricks` is a wrapper around the `databricks` CLI. It's the simplest way to get started without getting stuck with configuration. To learn more about Databricks Asset Bundles and customization, read [What are Databricks Asset Bundles](https://docs.databricks.com/en/dev-tools/bundles/index.html).
```

Comment:
I'd maybe more the "to learn more about DAB" to the very beginning of the page, where we say "This guide demonstrates a workflow for developing a Kedro Project on Databricks using Databricks Asset Bundles."